getFormsData.sh

```diff
@@ -50,26 +47,24 @@ URLtoPath () {
   fi
 }
 
-if [ ! -f "$SITES_PATHS_FILE" ]; then
-  # Convert each URL to path and save them in $SITES_PATHS_FILE
-  while IFS= read -r line; do 
-    echo "Extracting path from $line";
-    URLtoPath $line >> $SITES_PATHS_FILE;
-  done < $SITES_URLS_FILE
-fi
-
 # Generate the $DATA_CSV_FILE CSV file based on each path of $SITES_PATHS_FILE
```

Comment:
`SITES_PATHS_FILE` should be removed everywhere.

Reply:
Done